pxteditor/code-validation/getNestedChildBlocks.ts

```diff
@@ -0,0 +1,10 @@
+/// <reference path="../../localtypings/validatorPlan.d.ts" />
+
+import * as Blockly from "blockly";
+
+export function getNestedChildBlocks(parentBlock: Blockly.Block): Blockly.Block[] {
```

Comment:
Can we add a comment above this function explaining specifically which types of blocks this returns (for example, I'm unclear if it will return only immediate children or if it will also contain children of children)

Reply:
From the code I'd guess it's only immediate children, but would be nice to have documentation confirming

Reply:
Yeah, immediate children. I'll still add a comment, but I won't be able to have a visual in the comment, and I think a visual will help show what's happening. 

For example, if you have this if statement:
<img width="310" alt="image" src="https://github.com/microsoft/pxt/assets/49178322/b9926d52-c944-434b-ae03-d2db80383015">
`getNestedChildBlocks` will return the condition, the two variable assignments, and  the music block.